services/wallet/common/const.go

```diff
@@ -110,4 +116,5 @@ var AverageBlockDurationForChain = map[ChainID]time.Duration{
 	ChainID(EthereumMainnet): time.Duration(12000) * time.Millisecond,
 	ChainID(OptimismMainnet): time.Duration(400) * time.Millisecond,
 	ChainID(ArbitrumMainnet): time.Duration(300) * time.Millisecond,
+	ChainID(BaseMainnet):     time.Duration(400) * time.Millisecond,
```

Comment:
These values here are incorrect.
Should be:
- OptimismMainnet -> 2 seconds
- Arbitrum -> 0.25 seconds
- Base -> 2 seconds

Reply:
Updated